src/main/java/lux/fm/bookingservice/dto/booking/BookingRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package lux.fm.bookingservice.dto.booking;
+
+import java.time.LocalDate;
+
+public record BookingRequestDto(
+        Long accommodationId,
+        LocalDate checkIn,
+        LocalDate checkOut
+) {
```

Comment:
Add validations, NotNull, Positive something like this. Maybe also exist some annotations for Dates that before today or something like this